config/definitions/pot.js

```diff
@@ -29,7 +29,7 @@ const publicKeyURLs = [
 
 /** Context URLs. */
 const contextURLs = {
-    DataProduct: 'https://standards-ontotest.oftrust.net/v2/Context/DataProductOutput/Document/Signing/SignSpace/'
```

Comment:
because of this change, robottests should be also updated right?  here https://github.com/PlatformOfTrust/connector-signspace-with-tests/blob/b50728bea6530e28722ab4226c2b5c61a11afd39/robottests/connector_tests.robot#L65